tests/Counter.test.js

```diff
@@ -0,0 +1,28 @@
+import counter from '../src/modules/helpers.js';
+
+describe('Counter test', () => {
+  test('counter returns number of shows or comments', () => {
+    const comments = [{
+      item_id: 1,
+      username: 'John Doe',
+      comment: 'comment 1',
+    }, {
+      item_id: 1,
+      username: 'J B',
+      comment: 'comment 2',
+    }];
+
+    const shows = [{
+      item_id: 1,
+      name: 'show 1',
+      description: 'show 1 description',
+    }, {
+      item_id: 2,
+      name: 'show 2',
+      description: 'show 2 description',
+    }];
+
```

Comment:
- [ ] Great job adding tests for the counter functions, however, kindly ensure that you have tests for all edge cases, for instance, a scenario in which there are 0 items on the page (counter function should return 0)